packages/components/src/Helpers.purs

```diff
@@ -117,3 +122,17 @@ getMinStartDate nextDelivery = do
           soonestStart = adjust soonestDuration (DateTime.date now)
           byNextIssue = max <$> soonestStart <*> nextDelivery
       pure $ byNextIssue <|> soonestStart
+
+matchesRegex :: String -> String -> Boolean
```

Comment:
I can't say that I'm a fan of this function. Just using Regex directly is not that much different, this doesn't add that much. And this constructs a new Regex object for each call as well, presumably the use case is to use the same regex every time?

Reply:
The advantage of this is that it's concise and obvious, while the functions in the Regex module are a bit fiddly. But fine, it can be removed at the cost of a few extra lines and less clarity at the place where it was used. And then we can also precompile the regex object.

Reply:
Removed